provider/vsphere/environ_broker_test.go

```diff
@@ -62,11 +62,11 @@ func (s *environBrokerSuite) createStartInstanceArgs(c *gc.C) environs.StartInst
 	tools := setInstanceConfigTools(c, instanceConfig)
 
 	return environs.StartInstanceParams{
-		ControllerUUID: instanceConfig.Controller.Config.ControllerUUID(),
-		InstanceConfig: instanceConfig,
-		Tools:          tools,
-		Constraints:    cons,
-		Placement:      "zone=z1",
+		AvailabilityZone: "z1",
```

Comment:
Generally mandating AvailabilityZone as a single value feels like its going the wrong way.